src/com_tjnotifications/admin/sql/updates/mysql/2.0.2.sql

```diff
@@ -0,0 +1,58 @@
+-- Adding default values to the column of existing table
+
+ALTER TABLE `#__tj_notification_providers`
+	CHANGE `provider` `provider` varchar(100) NOT NULL DEFAULT '',
```

Comment:
add one SQL statement per column change

eg:

```
ALTER TABLE `#__jma_frequencies` CHANGE `created_by` `created_by` INT(11) NOT NULL DEFAULT 0;
ALTER TABLE `#__jma_frequencies` CHANGE `ordering` `ordering` INT(11) NOT NULL DEFAULT 0;
```

@divyachaudhari @ankush-maherwal 

AFAIK that's how Joomla wants it